docs/src/misc.rst

```diff
@@ -544,4 +544,7 @@ API
         process priority, the result will equal one of the `UV_PRIORITY`
         constants, and not necessarily the exact value of `priority`.
 
+        Setting PRIORITY_HIGHEST will only work for elevated user, for others it
```

Comment:
Should PRIORITY_HIGHEST and PRIORITY_HIGH be in backticks? It also might be worth making this a separate `note::` and emphasizing that this is a Windows thing.